packages/selection/src/index.js

```diff
@@ -0,0 +1,24 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+/* Components */
+export { default as LocaleProvider, LocaleContext } from './components/LocaleProvider';
+
+/* Hooks */
+export { default as useKeyboardFocus } from './hooks/useKeyboardFocus';
+export { default as useField } from './hooks/useField';
+export { default as useSelection } from './hooks/useSelection';
+
+/* Render-props */
+export { default as KeyboardFocusContainer } from './containers/KeyboardFocusContainer';
+export { default as FieldContainer } from './containers/FieldContainer';
+export { default as SelectionContainer } from './containers/SelectionContainer';
+
+/* Utils */
```

Comment:
It's not intuitive to me that we would find these here. A `container-utilities` package perhaps?

Reply:
With the way we "compose" the hooks/containers together with `useSelection` as the base, we will have to ensure that other hooks are using the correct utilities as well.  IdManger is a good example since it's global.

I would lean toward just matching the breakdown from react-components.